### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server_Modules_Screen Reader: Narrator is not reading the label names of all the controls in all the pages. 

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#86]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106
Screen Reader: Narrator, NVDA
**WCAG ID Violated:** WCAG1.3.1

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Edge; #Chrome; #Narrator; #NVDA; #WCAG1.3.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Server and press enter
5.	Tab till Modules and press enter
6.	Start Screen Reader (Narrator/NVDA).
7.	Validate that screen reader is reading the label names of all the controls.

**Actual Result:**
Narrator is not reading the label names of all the controls in all the pages. 

**Expected Result:**
Narrator should read the label names of all the controls in all the pages. 

**User Impact:**
Users with visual disabilities may not be aware of the label names if screen reader is not reading them.

**Attaching video for your reference:**
[Bug 86_Narrator_LabelNames.pptx](https://github.com/Microsoft/IIS.WebManager/files/2318373/Bug.86_Narrator_LabelNames.pptx)

**Comment by jhkimnew** [comment]
duplicate bug to https://github.com/Microsoft/IIS.WebManager/issues/63

**Comment by Roopesh-T** [comment]
**GitTags**: #A11ySev2;

**Comment by Yashi1201** [comment]
#Closed;
Manage.iis.net is deprecated. The "Manage IIS" button on the www.iis.net home page will be removed.
